Themes/default/languages/Packages.english.php

```diff
@@ -171,6 +171,7 @@
 $txt['package_modification_malformed'] = 'Malformed or invalid modification file.';
 $txt['package_modification_missing'] = 'The file could not be found.';
 $txt['package_no_zlib'] = 'Sorry, your PHP configuration doesn\'t have support for <strong>zlib</strong>. Without this, the package manager cannot function. Please contact your host about this for more information.';
+$txt['package_no_phar'] = 'Sorry, your PHP configuration doesn\'t have support for <strong>phar</strong>. Without this, the package manager cannot function. Please contact your host about this for more information.';
```

Comment:
Perhaps now that there is another "no support for x" string might be a good idea to make it generic, IE: 

"Sorry, your PHP configuration doesn\'t have support for <strong>{library}</strong>. Without this, {feature} cannot function. Please contact your host about this for more information."

This will also help mod authors too.

Reply:
well i see this not as task of this pr,
feeld free to do a own pr or to duplicate this pr (than i can close this),
where you add this.

Reply:
@MissAllSunday's idea is a good one, but if @albertlast would rather have that left to another PR, that's fine.

Reply:
Sure I'll add a generic string